clients/client-python/gravitino/api/types/types.py

```diff
@@ -123,7 +123,7 @@ def get(cls) -> "ShortType":
             return cls(True)
 
         @classmethod
-        def unsigned(cls):
+        def unsigned(cls) -> "ShortType":
```

Comment:
Can you help share with me what this syntax is?
I mean ... why the return type is quoted?

Reply:
```
    class ShortType(IntegralType):
        _instance: "ShortType" = None
        _unsigned_instance: "ShortType" = None
   ```
The type annotation ShortType is quoted because `_instance` is an attribute of the class that references ShortType, and the class itself isn't fully defined at the time this attribute is declared.

The return type annotations (-> "ShortType") are also quoted for the same reason: `ShortType` isn't fully defined when these methods are declared.

Reply:
If we remove the quote, there is a Pylance errors, and it won't pass the pylint check.